lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -0,0 +1,534 @@
+"""
+Process REDCap DETs that are specific to the
+COVID-19 Public Health Surveillance (SCAN) Project.
+"""
+import re
+import click
+import json
+import logging
+from uuid import uuid4
+from typing import Any, Callable, Dict, List, Mapping, Match, Optional, Union, Tuple
+from datetime import datetime
+from cachetools import TTLCache
+from id3c.db.session import DatabaseSession
+from id3c.cli.command.etl import redcap_det
+from id3c.cli.command.geocode import get_response_from_cache_or_geocoding
+from id3c.cli.command.location import location_lookup
+from seattleflu.id3c.cli.command import age_ceiling
+from .redcap_map import *
+from .fhir import *
+from . import race
+
+
+LOG = logging.getLogger(__name__)
+
+
+REVISION = 1
+
+REDCAP_URL = 'https://redcap.iths.org/'
+INTERNAL_SYSTEM = "https://seattleflu.org"
+
+PROJECT_ID = 20759
+REQUIRED_INSTRUMENTS = [
+    'screening',
+    'shipping_information',
+    'back_end_mail_scans',
+    'illness_questionnaire'
+    'nasal_swab_collection',
+    'post_collection_data_entry_qc'
+]
+
+
+@redcap_det.command_for_project(
+    "scan",
```

Comment:
I'd probably make this `scan-en` for English in anticipation of expanding to other languages. We can bundle all `scan-*` commands into this file as future work, and they can call the exact same command function.